crates/frontend/src/pages/dimensions.rs

```diff
@@ -50,6 +73,7 @@ pub fn dimensions() -> impl IntoView {
                 _ => Some(json!(function_name.replace('"', ""))),
             };
             let mandatory = row["mandatory"].as_bool().unwrap_or(false);
+            let dimesnion_name = row_dimension.clone();
```

Comment:
```suggestion
            let dimension_name = row_dimension.clone();
```
Would be better if this can be done.

Reply:
Yes , separated client_error logic and server_error logic